mathesar/tests/views/api/test_table_api.py

```diff
@@ -333,6 +332,13 @@ def test_table_type_suggestion(client, schema, engine_email_type):
     assert response_table == EXPECTED_TYPES
 
 
+def _check_columns(actual_column_list, expected_column_list):
+    # Columns will return an extra type_options key in actual_dict
+    # so we need to check equality only for the keys in expect_dict
+    for index, column_dict in enumerate(expected_column_list):
```

Comment:
Small nitpick, it might be nicer to iterate over a zip of the two lists.

Reply:
Agreed, I'll make the change. Thanks!